test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -258,7 +258,10 @@ def test_pytorch_amp(pytorch_training, ec2_connection, gpu_only, ec2_instance_ty
 @pytest.mark.integration("pt_s3_plugin_gpu")
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
-def test_pytorch_s3_plugin_gpu(pytorch_training, ec2_connection, gpu_only, ec2_instance_type, pt17_and_above_only):
+def test_pytorch_s3_plugin_gpu(pytorch_training, ec2_connection, gpu_only, ec2_instance_type):
+    _, framework_version = get_framework_and_version_from_tag(pytorch_training)
```

Comment:
Can we create a pytest fixture similar to `pt17_and_above_only` instead of defining the condition in every test?